### ISSUE TITLE
Multiple reminders

### ISSUE THREAD

**Issue by lachmanfrantisek** [fsobolev/timeswitch#58]
Hello!

Thanks for this simple but very helpful app!

Would it be possible to support multiple parallel countdowns?

I understand this will require significant changes in the user interface. Not sure if it helps, but Alarm and Timer view from Gnome Clock can be taken as an inspiration.

![image](https://user-images.githubusercontent.com/20214043/219654020-e3449a0c-d8a7-4585-87d8-578620e892b3.png)
![image](https://user-images.githubusercontent.com/20214043/219654251-f345c436-aea0-4b4d-bfef-0e875af9cf6b.png)

Or maybe an Android 12 alarm view (details accessible from the list view, no secondary window opened for details):

![Screenshot_20230217-133740](https://user-images.githubusercontent.com/20214043/219655792-25106a5a-bc04-441d-80c3-dcae1ca33363.png)

**Comment by fsobolev** [comment]
I need to consider it.  There are other features that would be nice to have, like running multiple commands (#51) or setting a timer for another day, but I considered them out of scope for Time Switch. And with this I'm not sure if I want to implement it in Time Switch or at some point in the future I will create another app that will be much more feature-rich.

**Comment by lachmanfrantisek** [comment]
@fsobolev , thank you for considering this! Would providing a few use cases help?

**Comment by fsobolev** [comment]
> Would providing a few use cases help?

I can easily imagine a situation when timers for multiple tasks running in parallel are required, like cooking for example. :)

**Comment by fsobolev** [comment]
Feel free to try convience me otherwise, but I think it would be better to implement in a separate task scheduler app for the reasons I explained in https://github.com/fsobolev/timeswitch/issues/60#issuecomment-1441558994
